### PR TITLE
update the README

### PR DESCRIPTION
**Pull request by depombo**

### CODE REVIEW

File: logo-dark.png
Comment (dfellis) [inline logo-dark.png:1]: This is a placeholder logo, right?

File: README.md [hunk 1]
```diff
@@ -1,15 +1,45 @@
...
+
+## Syntax
+
+The Marsha syntax is meant to be:
+- minimal and "obvious", but also discourage lax or incomplete information that could lead to unpredictable behavior
 - be mechanically parseable for syntax highlighting and quick feedback on correctness issues to the user
 - make it easy to define examples to reduce the probability of generating faulty code and allow generating tests that the application code can be tested against
 
-For now, only function and data structure definitions have been defined which allow for data mangling scripts to be created. The syntax is subject to change as Marsha is currently in an alpha state. What other elements to create will depend on the initial target audience and use case.
+The Marsha syntax looks a lot like markdown and is a mixture of English and mathematical notation. It has its own file format `.mrsh` and is used to define function(s) by providing an input, output (if not void), and the requirements described. The syntax is subject to change as Marsha is currently in an alpha state. If you have a legitimate use case for Marsha, please let us know.
```
Comment (aguillenv) [inline README.md:31]: maybe talk also about data types?

**Review by aguillenv — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -15,31 +15,71 @@ Marsha is a functional, higher-level, English-based programming language that ge
 
 ## Usage
 
-The Marsha compiler can be invoked via a CLI or `pip` module that can also be invoked in Jupyter notebooks.
+The Marsha compiler can be used to compile the syntax using a `pip` module via a terminal or Jupyter Notebook:
 
-<> TODO Demo
-
-<> TODO some instructions
+```bash
```
Comment (aguillenv) [inline README.md:20]: should we also add the make install + marsha <src> syntax?

**Review by aguillenv — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (depombo) [reply README.md:20]: I think we can start with this as it will be easier to consume since presumably they already have python installed

**Review by aguillenv — APPROVED** [review]

File: logo-dark.png
Comment (dfellis) [reply logo-dark.png:1]: Well, it looks like an N instead of an M for starters, but it's also *very* different from what we discussed.

**Pull request merged**